data/beta/field/public-body-account-classifications.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: public-body-account-classifications
+phase: beta
+register: public-body-account-classification
+text: National accounting classifications that a public body can have.
```

Comment:
can we come up with a better description please, maybe "that a public body can be assigned" or "that a public body can hold."

doesnt read right atm